### ISSUE TITLE
Parser error for boolean attributes without an explicit value

### ISSUE THREAD

**Issue by jvdl** [mdx-js/eslint-mdx#429]
### Initial checklist

- [X] I read the [support docs](https://mdxjs.com/community/support/)
- [X] I read the [contributing guide](https://mdxjs.com/community/contribute/)
- [X] I agree to follow the [code of conduct](https://github.com/mdx-js/.github/blob/HEAD/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Amdx-js&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

2.0.4

### Link to runnable example

https://github.com/jvdl/eslint-plugin-mdx-boolean-attributes

### Steps to reproduce

I was able to keep the reproduction very minimal and it only requires `eslint` and `eslint-plugin-mdx` together with some ESLint config and an MDX file to lint.

The ESLint configuration I used was

```js
module.exports = {
  overrides: [
    {
      files: ['*.mdx'],
      extends: 'plugin:mdx/recommended',
    },
  ],
  rules: {},
};
```
Beyond this the only other required items are:
- Create an MDX file with the content `<button disabled>Hi</button>`.
- Lint the file with the `eslint-plugin-mdx` plugin installed.
- Note the error `Parsing error: Expected value to be truthy`

There's a [minimal repro available](https://github.com/jvdl/eslint-plugin-mdx-boolean-attributes), and I've also [added a failing test in my fork of this repo](https://github.com/mdx-js/eslint-mdx/compare/master...jvdl:eslint-mdx:master).

### Expected behavior

The file is linted correctly.

Boolean attributes are truthy by virtue of their presence, rather than their explicitly assigned value. 

### Actual behavior

The linter throws an error when attempting to parse the file.

### Runtime

Node v16

### Package manager

npm v8

### OS

macOS

### Build and bundle tools

Other (please specify in steps to reproduce)

**Comment by JounQin** [comment]
Thanks for reproduction, I'll check soon.

**Comment by JounQin** [comment]
@jvdl Please help to test:

```shell
# yarn 1
yarn add https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-mdx https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-plugin-mdx
# yarn 2, 3
yarn add eslint-mdx@https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-mdx/_pkg.tgz eslint-plugin-mdx@https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-plugin-mdx/_pkg.tgz
# npm
npm i https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-mdx https://pkg.csb.dev/mdx-js/eslint-mdx/commit/c2bb0d9d/eslint-plugin-mdx
```

**Comment by jvdl** [comment]
Nice one @JounQin that seems to work perfectly!

**Comment by ransomledair** [comment]
I have observed this same issue for empty string props. e.g.
```
<MyComponent myProp="" />
```
results in
```
  0:0  error  Parsing error: Expected value to be truthy
```
from mdx eslint.
